README.md

```diff
@@ -24,14 +15,118 @@ Its meetings and membership are largely focused on the direction, design, and wo
+  * Repositories:
+    * **[Proposal]**: Multilateration of acoustic beacons
+  * References: 
+    *  [Mesh Navigation](https://github.com/uos/mesh_navigation)
+    *  [Fuse Localization](https://github.com/uos/mesh_navigation)
+    *  [MagNav](https://www.gpsworld.com/us-air-force-to-explore-navigating-with-magnetism/)
+
+* Perception
+  * Description: Packages for perception algorithms for maritime robotics.
+  * Motive: Since there are different sensors such as sonars, there is a need to process this data to obtain information. Moreover, it can be created some common structures detection.
+  * Repositories:
+    * **[Proposal]**: Common Maritime Environments Detections (buoys, valves, pipes...)
+    * **[Proposal]**: Fusion of sonar and camera data
+  * References: 
+    * Suggestions...
+   
+* Robots
+  * Description: Packages with maritime robots and structures.
+  * Motive: All the software being develop shall have not only unit tests but integration tests, so robots and structures shall be developed for this purporse. These shall include vcstool for specifying packages to be used with the robot as well as ros_ign_bridge to test it on simulation.
+  * Repositories
+    * **[Proposal]** Hover type UW vehicle
+    * **[Proposal]** Torpedo shaped UW vehicle
+    * **[Proposal]** USV
+    * **[Proposal]** Intervention UW vehicle 
+  * References: 
+    *  Intervention UW sample [RexROV2](https://uuvsimulator.github.io/packages/rexrov2/intro/)
+    *  Torpedo Shaped UW vehicle [lrauv](https://github.com/osrf/lrauv/)
+   
+* Simulation
+  * Description: Documentation and index for maritime robotics Ignition plugins.
+  * Motive: There is no need to mix ros code with simulation, also there is already some maritime robotics development in Ignition, so is a good thing to contribute to ign-plugin repo instead of creating other in this organization.
+  * Repositories
+    * **[Proposal]** Maritime Robotics Ignition Plugins Index
+    * **[Proposal]** Wave Simulation (repo shall be in ign-plugins)
+    * **[Proposal]** Acoustics Simulation (repo shall be in ign-plugins) (Include salinity and temperature effects)
+    * **[Proposal]** Radar Simulation (repo shall be in ign-plugins)
+    * **[Proposal]** Sonar Simulation (repo shall be in ign-plugins)
+  * References: 
+    *  [ASV Wave Simulator](https://github.com/srmainwaring/asv_wave_sim/tree/feature/fft_waves)
+
+* Tools
+  * Description: Extra category for packages that do something specific that don't fit in any other category
+  * Motive: There might be some tools that can be developed for maritime robotics that doesn't fit another category, such as geographic information convertion. 
+  * Repositories
+    * **[Proposal]** Geographic convertions
+  * References: 
+    *  Suggestions...
 
 ### Standards for subprojects
```

Comment:
Some of the items on this list are pretty ambitious and require a lot of maintenance effort. I don't know how the WG will enforce this, or how strict it should be. It can be difficult to check all these boxes.

Lower sections (because I can't add separate comments to existing lines not in this PR) mention unanimous approvals from Approvers. How many Approvers are there going to be? Unanimous might not be the best strategy - what if someone is on leave for a month? Frankly, I don't know how other WGs handle this, but there should be flexibility built in for such situations. Maybe a specific number of percentage of Approvers.

Reply:
Sure! What do you think it would be the best strategy? I though about throwing here every idea that we may develop in the future, even if it is 2 or 3 years later... @arjo129 commented about having a road-map to better clarify when we are going to tackle each problem.

About the approval, I changed it to 2, what do you think? I think that if at least two approvers agree that the PR was good, then it is enough.

Reply:
I see. I think the ones requiring the most effort on this list are setting up things with the build farm, making sure things are green (this can take a lot of engineering time), and making sure somebody is doing the releases. While the setup is a one-time effort, the others are routine maintenance and require people to commit regularly.

Other things like maintaining backlog, tickets receiving prompt responses, these are active development tasks that might benefit from triaging during meeting time. (Just one idea I heard some WGs do. May or may not apply to this WG.)

It'd be good to gauge commitment levels at the next meeting, to see how ready people are to commit to some of these. Of course, there isn't much work to do maintenance-wise before people figure out who's developing what features, so this section could be a WIP for now, until people figure out what works and modify this.

My original comment just wanted to point out that this list of standards can take a lot of effort, and some of them might be marked as tentative, or maybe there should be a note saying it's best effort, or up to modifications until a certain point, etc. Basically in the beginning it should be easy to change, without being abused later to turn into, if the WG can't tackle something, then just go change the charter, if you know what I mean.

Re road map: Yeah, we were talking about road maps as the next steps for the community in general, because there has already been a few meetings where people come and talk about what they do, but nothing moves on from there. A road map doesn't need to have strict calendar dates, if the WG isn't quite ready for that. Another way is to have an effort estimate table/list - whether something will take days, weeks, months. Yet another way is to have a table of "tentative assignments," if you think people are ready to commit to developing certain features at this time. That could be good if they are already doing something and are willing to contribute to open source.

Re approval number: Yeah, I think 2 should be fine. We use 2 for rosdistro rosdep approvals, which is kinda important. If it's good enough for that, it should be good enough for this.

Reply:
Regarding the ROS build farm, setting CI and such, I don't have experience with that. Do you have a package and links so I can better understand this?

@arjo129 has created the issue tracking and the project for better organization

I didn't understand much your comment about the charter. Do you think that is better if we reduce the scope and change it as we think we can tackle something instead of listing everything we might be able to do?

Reply:
rosdistro is a good starting point, if that is the desired outcome. I believe some people (UNH) in the monthly meetings have experience Bloom releasing their packages to rosdistro and getting them distributed.
https://github.com/ros/rosdistro/blob/master/CONTRIBUTING.md
I'm not on regular rosdistro duty anymore as of couple months ago, but I can try to answer questions based on what I know.

Yeah the language of the charter is pretty strict. If it's in the charter, then it seems to suggest that it's something that should be done. If it isn't done, then it shouldn't be in the charter - it should be in an issue to be later added to the charter once resources are found. The charter shouldn't be a wish list, in my opinion. People would read the charter with the understanding that this is the standard of the WG and everything is carried out. It should include what's actually being done.